docs/sphinx/sections/api/apidocs/graphs.rst

```diff
@@ -14,3 +14,14 @@ The replacement for :py:class:`composite_solid` / :py:class:`CompositeSolidDefin
 .. autoclass:: GraphIn
 
 .. autoclass:: GraphOut
+
+
+Dependencies and aliases
```

Comment:
nit: im not sure the best way to communicate it - but these artifacts are really just for manually creating the dep structure instead of using a composition function. Maybe "manual dependency structures" or "explicit dependencies" or something in that direction?

Reply:
Good point. I'll go with "explicit dependencies".